### PR TITLE
Fixed CA1716 - Identifiers should not match keywords

### PR DESCRIPTION
**Pull request by scowalt**
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1716?view=vs-2019

This is a particularly nice rule since we're integrating our code into C++ codebases

### CODE REVIEW

File: Source/Libraries/CorruptCore/IRealTime.cs [hunk 1]
```diff
@@ -8,7 +8,6 @@ namespace RTCV.CorruptCore
 
     public interface IRealTime
     {
-        event EventHandler<RealTimeEventArgs> Step;
```
Comment (scowalt) [inline Source/Libraries/CorruptCore/IRealTime.cs:11]: This builds but I don't have context for this.

Comment (NarryG) [reply Source/Libraries/CorruptCore/IRealTime.cs:11]: Some plugins subscribe to it, none that are in public repos in the moment though.

**Review by NarryG — CHANGES_REQUESTED** [review]
That Step needs to be left in since some (currently unreleased) plugins subscribe to it

File: Source/Libraries/CorruptCore/IRealTime.cs (hunk 1 above)
Comment (scowalt) [reply Source/Libraries/CorruptCore/IRealTime.cs:11]: Can I rename it to satisfy `CA1716` and break the private plugins? Or should I make an exception here?

Comment (NarryG) [reply Source/Libraries/CorruptCore/IRealTime.cs:11]: They're just not done really. All the other changes already completely broke them all so yeah go ahead. I've long accepted that lots of stuff is going to need fixes as you refactor. It's much less effort to recompile them than it is to deal with a messy codebase so go feel free to go forwards.

Comment (scowalt) [reply Source/Libraries/CorruptCore/IRealTime.cs:11]: Done

Comment (scowalt) [reply Source/Libraries/CorruptCore/IRealTime.cs:11]: Ah, this is actually being used in Bizhawk. I'll need to make a companion fix here.

**Pull request merged**